### PR TITLE
feat: document deployments and releases

### PR DESCRIPTION
**Pull request by bethesque**

### CODE REVIEW

**Comment by bethesque** [comment]
https://github.com/pact-foundation/docs.pact.io/blob/feat/deployments-and-releases/website/blog/2021-07-04-why-we-are-getting-rid-of-tags.md

File: website/blog/2021-07-04-why-we-are-getting-rid-of-tags.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  We all like tools that help us use them better. The addition of branches and environment support allows the Broker to detect how far along a user is in getting a Pact supported CI/CD workflow set up (eg. have they set up any environments, have they recorded any deployments). The output from the new contract and verification publishing APIs now includes "next steps", which tell a user what technical tasks need to be completed next to achieve a fully Pact supported workflow.
+
+As time goes on, explicit branch and environment support will continue to help us make Pact easier and better to use. Death to tags! Long live branches and environments!
+
+## How do I migrate from tags to branches and environments?
+
+* Upgrade to the latest version of the Pact Broker
+* Upgrade to the latest version of your Pact clients
+* Set the "main branch" property for each pacticipant in the Pact Broker, using the `create-or-update-pacticipant` command.
+* Create an environment resource for each of your deployment/release environments using the Pact Broker CLI.
```
Comment (PauloGoncalvesBH) [inline website/blog/2021-07-04-why-we-are-getting-rid-of-tags.md:59]:
```suggestion
* Create an environment resource for each of your deployment/release environments using the `create-environment` command.
```

**Comment by YOU54F** [comment]
@bethesque does anything in this branch want saving?

the name suggests there is something useful there `feat/deployments-and-releases` :)

**Comment by bethesque** [comment]
I don't know what happened to this branch. It got totally effed. I'm pretty sure I rescued the important content from it though. If not, it's probably due for a re-write anyway.

**Pull request closed without merging**